python/utils/run_benchmarks.py

```diff
@@ -180,71 +148,84 @@ def parseCLI(argv):
                         help='Automatically do the csr and transposed '
                         'transformations. Default is 0, switch to another '
                         'value to enable')
-    parser.add_argument('--times_only', action="store_true",
-                        help='Only output the times, no table')
     parser.add_argument('--delimiter', type=str, choices=["tab", "space"],
                         default="space",
                         help='Delimiter for csv files (default is space)')
+    parser.add_argument('--update_results_dir', type=str,
+                        help='Add (and compare) results to the dir specified')
+    parser.add_argument('--update_asv_dir', type=str,
+                        help='Add results to the specified ASV dir in ASV '
+                        'format')
+    parser.add_argument('--report_cuda_ver', type=str, default="",
+                        help='The CUDA version to include in reports')
+    parser.add_argument('--report_python_ver', type=str, default="",
+                        help='The Python version to include in reports')
+    parser.add_argument('--report_os_type', type=str, default="",
+                        help='The OS type to include in reports')
+    parser.add_argument('--report_machine_name', type=str, default="",
+                        help='The machine name to include in reports')
+
     return parser.parse_args(argv)
 
 
 def getAllPossibleAlgos():
-    class fakeArgs:
-        def __getattr__(self, a): return None
-    return list(getAlgoData(None, {"src": 0, "dst": 0}, fakeArgs()).keys())
-
+    return list(getBenchmarks(nop, nop, nop).keys())
 
 ###############################################################################
 if __name__ == "__main__":
     perfData = []
     args = parseCLI(sys.argv[1:])
-    delimiter = {"space": ' ', "tab": '\t'}[args.delimiter]
 
+    # set algosToRun based on the command line args
     allPossibleAlgos = getAllPossibleAlgos()
```

Comment:
How would you run the same algo twice with different parameters? Let's say we want to benchmark SSSP a few times changing the `source`. Does this involve running the whole script twice using `parseCLI`or is there a smarter way?

Reply:
I think right now you'd run the script multiple times with different `--source` like you said. You can also have it run only the SSSP algo using the `--algo` arg, so that way you wouldn't need to re-run every algo just to get multiple SSSP runs.  I think I can improve this use case in a future PR though.